resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentServiceImpl.java

```diff
@@ -598,7 +598,7 @@ public ResidentUpdateResponseDTO reqUinUpdate(ResidentUpdateRequestDto dto) thro
 			regProcReqUpdateDto.setMachineId(machineId);
 			regProcReqUpdateDto.setIdentityJson(dto.getIdentityJson());
 			List<ResidentDocuments> documents = dto.getDocuments();
-			byte[] decodedDemoJson = CryptoUtil.decodeBase64(dto.getIdentityJson());
+			byte[] decodedDemoJson = CryptoUtil.decodePlainBase64(dto.getIdentityJson());
```

Comment:
if ResidentUpdateService.java is doing url safe, the same should be used here since this dto is loaded from the same request object